packages/terra-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -178,6 +178,22 @@ function WorklistDataGrid(props) {
   const gridReceivedFocus = useRef(false);
   const gridHasFocus = document.getElementById(`${id}-worklist-data-grid-container`)?.contains(document.activeElement);
 
+  const makeWorklistDataGridColumns = (column) => ({
+    ...column,
+    isResizable: column.isResizable !== false,
+    isSelectable: column.isSelectable !== false,
```

Comment:
It looks like this might be duplicated in DataGrid.  If so, I think we have to choose one approach.  If we are going to update in the subcomponents, which is ok, we would remove the logic from the DataGrid.